NickvisionTagger.GNOME/Helpers/Help.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace NickvisionTagger.GNOME.Helpers;
+
+/// <summary>
+/// Helper class for help docs
+/// </summary>
+public static class Help
+{
+    /// <summary>
+    /// Get URL for given help page
+    /// </summary>
+    /// <param name="pageName">Help page name</param>
+    /// <returns>URL to either yelp or web page</returns>
+    public static string GetHelpURL(string pageName)
+    {
+        if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("SNAP")))
+        {
+            return $"help:tagger/{pageName}";
+        }
+        using var linguasStream = Assembly.GetCallingAssembly().GetManifestResourceStream("NickvisionTagger.GNOME.LINGUAS");
+        using var reader = new StreamReader(linguasStream!);
+        var linguas = reader.ReadToEnd().Split(Environment.NewLine);
+        var lang = "C";
+        if (linguas.Contains(CultureInfo.CurrentCulture.Name.Replace("-", "_")))
```

Comment:
I'd wrap this and the else all in an if statement checking if the culture is not invariant nor en-US because in that case it will always be C so no need to waste time iterating the linguas and checking